sdk/cognitivelanguage/Azure.AI.Language.Conversations/CHANGELOG.md

```diff
@@ -3,8 +3,13 @@
 ## 1.0.0-beta.4 (Unreleased)
 
 ### Features Added
```

Comment:
### Features Added
* Conversation issue summarization task (Long-running operation)
* Conversation PII extraction task (Long-running operation)

### Breaking Changes
* Client now uses python dictionaries for method parameters and results instead of classes.
* Many input and result parameter name changes in `analyze_conversation()` method